mtps-run-tests

```diff
@@ -211,7 +211,7 @@ for nevra in $nevras_in_repo; do
     fi
     if [ -n "$SELINUX" ] && rpm --quiet -q audit; then
         selinux_status=0
-        ausearch -m avc,user_avc,selinux_err,user_selinux_err -ts "$START_DATE" "$START_TIME" 2>&1 | grep -s -o 'no matches' >/dev/null 2>&1 || selinux_status=1
+        ausearch --format raw -m avc,user_avc,selinux_err,user_selinux_err -ts "$START_DATE" "$START_TIME" 2>&1 | sed -e '/received policyload notice/d' | grep -s -o '^' && selinux_status=1
```

Comment:
I am a bit puzzled by this part:
`| grep -s -o '^'`

What does that do in this context?

Reply:
1. sed command removes any lines with `received policyload notice`
2. grep - if there is any lines are left, then selinux falied

`grep -s -o '^'` -- is a tets for if there is at least 1 line
There should not be lines if selinux is OK.